### PR TITLE
Concurrent featured events

### PR DESCRIPTION
**Pull request by lazydevorg**
* Featured events now can be more than one.
* A `Read more` button has been added to the featured events to show the details about the speakers.
* An optional event's description has been added.

### CODE REVIEW

File: shared/components/Talks/index.js [hunk 1]
```diff
@@ -14,9 +25,8 @@ const Talks = ({ talks }) => {
           const speakerProps = pathOr({}, ['speakers', 0], talk);
           return (
             <div className="Talk" key={index}>
-              <Speaker {...speakerProps} />
-              <h4 className="Speaker__title">{talk.title}</h4>
-              <p className="Speaker__summary">{talk.summary}</p>
+              <Speaker {...speakerProps} collapsed={collapsed} />
+              {talkDetails(talk, collapsed)}
```
Comment (JoeStanton) [inline shared/components/Talks/index.js]: why don't you do `!collapsed && <TalkDetails talk={talk}>`. I think its a bit weird to have functions that return markup but aren't components... but just my opinion, doesn't really matter.

File: shared/components/NextCommunityEvent/index.js [hunk 2]
```diff
@@ -42,46 +42,79 @@ export function getHeaderText(startDateTime, endDateTime) {
...
+    super(props);
+    this.state = { collapsed: true };
+  }
+
+  expandInfo(e) {
+    this.setState({ collapsed: false });
+    e.preventDefault();
+  }
+
+  readmoreButton() {
```
Comment (asavin) [inline shared/components/NextCommunityEvent/index.js]: This one is a good candidate for a separate component. It's ok to have small components.

File: shared/components/NextConferenceEvent/index.js [hunk 3]
```diff
@@ -104,5 +104,6 @@ const NextConferenceEvent = ({ calendarURL, speakers, tickets }) => (
 NextConferenceEvent.propTypes = {
   calendarURL: PropTypes.string,
   speakers: PropTypes.arrayOf(speakerType),
+  tickets: PropTypes.arrayOf(PropTypes.shape(ticketType)),
 };
```
Comment (grahammendick) [inline shared/components/NextConferenceEvent/index.js]: Don't think you need this line? You've not changed the NextConferenceEvent component and I think it generates a props warning when viewing the conference page

**Comment by grahammendick** [comment]
There are a couple of items that don't match the designs:
* Before the Read More is clicked the Speaker Info is below the image but it should be to the right
* In Tablet screen mode, the Speakers shift slightly when Read More is clicked

**Comment by lazydevorg** [comment]
@grahammendick it has been decided after to not move the speakers' info from right to bottom. I think we decided to leave them on the bottom.

**Comment by samsmith1983** [comment]
The August React London Meetup event has ended. But it says 'tickets now sold out' on the page, when it should say 'this event has ended. watch the video here'

**Comment by lazydevorg** [comment]
@samsmith1983: I've fixed it. Have a look please

**Comment by samsmith1983** [comment]
Another issue: date is not showing when event is set to `PRE_RELEASE` and ticket release date is set

**Comment by samsmith1983** [comment]
The 'read more' button does not appear to be working on Android Chome 6.

**Comment by samsmith1983** [comment]
Another issue: This is what the page looks like in iOS9 Safari: 
screen shot 2016-12-23 at 10 53 43

**Comment by lazydevorg** [comment]
> The 'read more' button does not appear to be working on Android Chome 6.
I've tried and it seems to work. @samsmith1983 can you give more details about the device you used?

**Comment by samsmith1983** [comment]
@lazydevorg this is on a Samsung Galaxy S7, via browserstack. I'll grab a couple of physical devices to confirm now.

**Comment by samsmith1983** [comment]
The 'read more' issue was due to 'safe mode' being enabled on the test device. All good now.

**Comment by lazydevorg** [comment]
> Another issue: This is what the page looks like in iOS9 Safari:

@samsmith1983 can you please create an issue for this?
It was an already existing problem and it makes sense to work on it separately.

**Comment by samsmith1983** [comment]
@lazydevorg yep, didn't notice this was happening before. Will create an issue now.

**Comment by samsmith1983** [comment]
It's actually already logged under #276

**Comment by samsmith1983** [comment]
OK so the 'read more' issue appears to only happen with Javascript is disabled. Need to confirm whether this is a blocking issue or not.

**Pull request merged**